src/toaster/useToaster.ts

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import toaster from './toaster';
+import { ToastContainerProps } from './ToastContainer';
+import { useCustom } from '../utils';
+
+const useToaster = () => {
+  const { toasters } = useCustom();
+
+  return {
+    push: (message: React.ReactNode, options: ToastContainerProps = {}) => {
+      if (toasters) {
+        return toasters?.current.get(options.placement || 'topCenter')?.push(message);
```

Comment:
第一个 `?` 是多余的

Reply:
这里的api是不是应该改成
```suggestion
        toasters.push(message, options);
```
现在的api可能出现拿不到 `container` 导致消息不展示